appveyor.yml

```diff
@@ -24,6 +24,7 @@ install:
   - IF /I "%PLATFORM%" == "x64" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
   - IF /I "%PLATFORM%" == "x86" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
   - npm ci
+  - npm run update-crosswalk
```

Comment:
Great!  Now set lines 5 thru 7 of `appveyor.yml` to
```python
    - nodejs_version: 18
    - nodejs_version: 20
    - nodejs_version: 21
```
to match https://nodejs.org/en/about/previous-releases#release-schedule and your tests should be green.  ✅

Reply:
The tests would be green because of the newer Node versions, but are you sure I should remove the older versions from the tests?

Reply:
Well, it did not pass for Node 20 and 21.

Reply:
looks like there's an explanation for that here: https://github.com/mapbox/node-pre-gyp/pull/712/files#diff-92ab9a36df5d8e9f7076f2fdec59492d1ac2d9cf27ea046767a7fc4d542ef3dcR11